apps/block_scout_web/lib/block_scout_web/views/api/v2/address_view.ex

```diff
@@ -112,14 +113,18 @@ defmodule BlockScoutWeb.API.V2.AddressView do
         "has_beacon_chain_withdrawals" => Counters.check_if_withdrawals_at_address(address.hash, @api_true)
       })
 
-    if Enum.empty?(implementations) do
-      extended_info
-    else
-      Map.merge(extended_info, %{
-        "proxy_type" => proxy_type,
-        "implementations" => implementations
-      })
-    end
+    result =
+      if Enum.empty?(implementations) do
+        extended_info
+      else
+        Map.merge(extended_info, %{
+          "proxy_type" => proxy_type,
+          "implementations" => implementations
+        })
+      end
+
+    result
+    |> chain_type_fields(%{address: creation_transaction && creation_transaction.from_address, field_prefix: "creator"})
```

Comment:
This also could be an option

```suggestion
    if Enum.empty?(implementations) do
      extended_info
    else
      Map.merge(extended_info, %{
        "proxy_type" => proxy_type,
        "implementations" => implementations
      })
    end
    |> chain_type_fields(%{address: creation_transaction && creation_transaction.from_address, field_prefix: "creator"})
```

Reply:
IMHO current version is more readable